### PR TITLE
chore: Show chatwoot_edition in Instance details

### PR DESCRIPTION
**Pull request by memorycancel**
To check the edition anytime in admin dashboard, Showing as below:

image

### CODE REVIEW

File: app/controllers/super_admin/instance_statuses_controller.rb [hunk 1]
```diff
@@ -5,6 +5,17 @@ def show
...
+
+  def chatwoot_edition
+    is_enterprise = if ChatwootApp.enterprise?
+                      'Enterprise'
+                    else
+                      'Community'
+                    end
+    is_custom = ChatwootApp.custom? ? 'Custom' : ''
+    @metrics['Chatwoot edition'] = "#{is_enterprise} #{is_custom}"
   end
```
Comment (sojan-official) [inline app/controllers/super_admin/instance_statuses_controller.rb:20]:
```suggestion
  def chatwoot_edition
    @metrics['Chatwoot edition'] = case
    when ChatwootApp.enterprise?
      'Enterprise'
    when ChatwootApp.custom?
      'Custom'
    else
      'Community'
    end
  end

```

**Review by sojan-official — COMMENTED** [review]
Minor refactoring suggestions 
since we are only considering 3 cases
- community
- enterprise 
- custom

File: app/controllers/super_admin/instance_statuses_controller.rb (hunk 1 above)
Comment (memorycancel) [reply app/controllers/super_admin/instance_statuses_controller.rb:20]: Make sense, Enterprise custom and Community custom are both custom. It's more simple, thanks! suggestion applied, PTAL.

**Comment by memorycancel** [comment]
> Minor refactoring suggestions since we are only considering 3 cases
> 
>     * community
> 
>     * enterprise
> 
>     * custom

Make sense, Enterprise custom and Community custom are both custom. It's more simple, thanks! suggestion applied, PTAL.

File: app/controllers/super_admin/instance_statuses_controller.rb [hunk 2]
```diff
@@ -5,6 +5,17 @@ def show
     sha
     postgres_status
     redis_metrics
+    chatwoot_edition
+  end
+
+  def chatwoot_edition
+    @metrics['Chatwoot edition'] = if ChatwootApp.enterprise?
+                                     'Enterprise'
+                                   elsif ChatwootApp.custom?
```
Comment (pranavrajs) [inline app/controllers/super_admin/instance_statuses_controller.rb:14]: What is custom here? @vishnu-narayanan @sojan-official

**Pull request merged**